tests/phpunit/tests/media.php

```diff
@@ -3798,6 +3798,61 @@ function() use ( &$result ) {
 
 		$this->assertFalse( $result );
 	}
+
```

Comment:
The two tests here look great, but in addition, now that we have the global variable it would be good to add two more (simpler) tests for `wp_print_image_mime_fallback_script()`:
* One where the global is set to `false`, then calling `wp_print_image_mime_fallback_script()`. It shouldn't print anything.
* One where the global is set to `true`, then calling `wp_print_image_mime_fallback_script()`. It should print the script.

You could use an output buffer in both tests.

Those would be more like actual unit tests, so with that we would have two unit tests and the two integration tests below. 👍

Reply:
@felixarntz I have added two new tests but it's generate wired issue in `PHPUnit Tests / 5.6 slow tests on ubuntu-latest`

1) Tests_Media::test_wp_print_image_mime_fallback_script_return_fallback_script
copy(/var/www/src/wp-includes/js/wp-image-mime-fallback-loader.js): failed to open stream: No such file or directory

Don't replicate same issue in my machine. Do you know how to fix this?

cc. @SergeyBiryukov